src/pages/InteractiveAPI/InteractiveAPI.vue

```diff
@@ -93,6 +93,10 @@ query { hello }
       document.getElementById('graphiql')
     )
   },
+  beforeDestroy() {
+    const cdnScripts = ['react', 'react-dom', 'graphiql-js']
+    cdnScripts.forEach(script => document.getElementById(script).remove())
```

Comment:
Is there a downside to removing these every time the component is destroyed? My understanding is that these scripts can be fairly large requests.

Reply:
they don't seem comparatively huge,  235kb over network/875kb uncompressed. Also, this ensures we're getting the most recent versions (not likely an issue for navigating between pages, but could be if we were to check the cache for a file instead)

Reply:
Hm ok - that seems fine.